Package.swift

```diff
@@ -0,0 +1,86 @@
+// swift-tools-version:5.4.0
+
+import PackageDescription
+
+let package = Package(
+    name: "Ably",
+    platforms: [
+        .macOS(.v10_11),
+        .iOS(.v9),
+        .tvOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "Ably",
+            targets: ["Ably"]
+        ),
+    ],
+    dependencies: [
+        .package(name: "msgpack", url: "https://github.com/rvi/msgpack-objective-C", from: "0.4.0"),
+        .package(name: "AblyDeltaCodec", url: "https://github.com/ably/delta-codec-cocoa", .branch("main"))
+    ],
+    targets: [
+        .target(
+            name: "Ably",
+            dependencies: [
+                .byName(name: "msgpack"),
+                .byName(name: "AblyDeltaCodec")
+            ],
+            path: ".",
+            exclude: [
+                "Info-tvOS.plist",
+                "Info-macOS.plist",
+                "Info-iOS.plist",
+                "Ably-SoakTest-App",
+                "Ably-SoakTest-AppUITests",
+                "Spec",
+                "Products",
+                "Scripts",
+                "fastlane",
+                "Examples",
+                "Carthage",
+                "include/SocketRocketAblyFork",
+                "Makefile",
+                "test-resources",
+                "ably-common/test-resources",
+                "Ably.xcconfig",
+                "Cartfile",
+                "Cartfile.private",
+                "Cartfile.resolved",
+                "Gemfile",
+                "Gemfile.lock",
+                "Ably.podspec",
+                "Version.xcconfig",
+                "ably-common/protocol/README.md"
+            ],
+            sources: [
+                "Source",
+                "SocketRocket"
+            ],
+            resources: [
+                .copy("COPYRIGHT"),
```

Comment:
What are the benefits of  specifying these `.copy`, I didn't see any warnings or issues when I did not include any `resources`.  Reading the [docs](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#resource), I'm still not sure of the benefits.

All the resources in project root directory seem to copy just fine without specifying anything.

Reply:
My explanation on video https://s.tape.sh/SQU1HYd8

Reply:
Okay, I think the warnings you are getting is because you are setting path of your target to the be root of the project (`path: "."`). We can fix this in a separate PR and we can remove these lines in the future.

For example, SwiftLint is Swift Package, and has a lot of resources in the root folder, however, it doesn't not need to declare any resources.